packages/component-library/draft/Kaizen/Modal/Modal.elm

```diff
@@ -243,14 +278,20 @@ mapDuration duration =
 
 
 type Variant msg
-    = Generic (List (Html msg)) Size
+    = Generic (List (Html msg)) ( Float, Float )
+    | Confirmation ConfirmationType (ConfirmationConfig msg)
```

Comment:
what are the benefits of having confirmation type outside confirmation config?
would putting inside simplify things?

Reply:
No real benefits, it could potentially live inside the config!